guides/doc-Managing_Security_Compliance/master.adoc

```diff
@@ -31,8 +31,8 @@ include::common/modules/ref_supported-scap-versions.adoc[leveloffset=+2]
 
 include::common/modules/ref_compliance-policy-deployment-options.adoc[leveloffset=+1]
 
-ifndef::satellite[]
-include::common/modules/proc_installing-openscap-plugin.adoc[leveloffset=+1]
+ifndef::foreman-deb[]
```

Comment:
This guide isn't in the navigation for `foreman-deb`. Not sure hiding this module is necessary when OpenSCAP and subsequently the whole guide isn't available for `foreman-deb`.